05/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/global.css">
+    <link rel="stylesheet" href="css/mobile.css">
+    <link rel="stylesheet" media="(min-width: 800px)" href="css/desktop.css">
+    <title># `#05` HTML i CSS: Responsywność </title>
+</head>
+<body>
+    <header class="row header">
+        <nav class="nav container">
```

Comment:
Można by było użyć też modyfikatora np. `<nav class="nav container container--nav">`

Reply:
Dla pozostałych znaczników również należy wprowadzić zmiany w nazwach klasy? Czyli np. <section class="container main container--main" itd? 

![Zrzut ekranu 2022-01-01 o 14 40 03](https://user-images.githubusercontent.com/80978376/147851916-47b2e180-e90b-4e6c-8c8c-94703fe49005.png)

Reply:
Nie jest to konieczne, ale jak pisałem wyżej jest to alternatywne rozwiązanie. Wtedy faktycznie dla pozostałych warto byłoby to zrobić ;)